src/components/MangaComponent.vue

```diff
@@ -1,20 +1,28 @@
 <script setup lang="ts">
-import { ref } from "vue";
+import { ref, watch } from "vue";
 import { ArrowLeftCircleIcon, ArrowRightCircleIcon } from "@heroicons/vue/24/solid";
+import type { Book } from "@/types/types";
 
 interface Props {
   mangaId: string;
+  bookDetail: Book;
 }
 const props = defineProps<Props>();
 const page = ref(1); // 2*page, 2*page+1ページが写し出されている
 const lastPage = ref(88);
 const path0 = "../../contents";
 const mouseOnLeft = ref(false);
 const mouseOnRight = ref(false);
+watch(
+  () => props.bookDetail,
+  (bookDetail) => {
+    lastPage.value = bookDetail.totalPages;
+  },
+);
```

Comment:
これってこれでは駄目ですか？(意図を読み取れなかった)
```suggestion
const lastPage = computed(bookDetail.totalPages);
const path0 = "../../contents";
const mouseOnLeft = ref(false);
const mouseOnRight = ref(false);
```

Reply:
それで大丈夫です（単にcomputedがないとAPI取得前の値になるし、どう書けばいいかわからずに変な感じになってました）

Reply:
これ適用せずにマージしちゃった、別のPRで適用しておきます